src/main/java/com/example/springBoot_hibernate_ferrocarriles/controller/LocomotoraController.java

```diff
@@ -0,0 +1,53 @@
+package com.example.springBoot_hibernate_ferrocarriles.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.springBoot_hibernate_ferrocarriles.model.Locomotora;
+import com.example.springBoot_hibernate_ferrocarriles.service.LocomotoraService;
+
+@RestController
+public class LocomotoraController {
+	@Autowired
+	LocomotoraService locomotoraService;
+
+	public LocomotoraController() {
+		super();
+	}
+	
+	@GetMapping("/locomotora")
+	public List<Locomotora> getAllLocomotora(){
+		return locomotoraService.getAllLocomotora();
+	}
+	
+	@GetMapping("/locomotora/{locomotoraId}")
+	public Locomotora getLocomotoraById(@PathVariable("locomotoraId")Long locomotoraId) {
+		return locomotoraService.getLocomotoraById(locomotoraId);
+	}
+	
+	@PostMapping("/locomotora")
+	public int addLocomotora(@RequestBody Locomotora locomotora){
```

Comment:
Consider to return ResponseEntity object instead of primitives types(boolean, int, etc).
The method `addOrUpdateLocomotora` in `LocomotoraService` could return a Locomotora entity, hence the controller too.